Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,37 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
```

Comment:
I reckon we leave this `unless` as-is and then just `return` afterwards so it can skip all the stuff below in that case.